Automattic/jetpack
# Instant Search: Hide search options separators from screen readers.

Moves the separator "•" from CSS into the DOM and sets the `aria-hidden` attribute so that the separators are ignored by screen readers.

#### Changes proposed in this Pull Request:


- Removes the CSS that added the "•" separators.
- Adds them inline to the HTML.
- Updated CSS to maintain styling.
- Adds `aria-hidden` attribute to hide from screen readers.

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






1. Load the Instant Search overlay.
2. Inspect the "•" separators.
3. Confirm they all have `aria-hidden="true"` set.
4. Test in Firefox using the Accessibility inspector.
5. Confirm the separators are not visible there.

Screenshot:

![NoSeps](https://user-images.githubusercontent.com/40267301/180377673-1ce58c1d-327c-4bf6-9978-378494c97141.png)
